ts-eks-helm/klotho.yaml

```diff
@@ -0,0 +1,17 @@
+app: ts-eks-helm
+provider: aws
+path: .
+out_dir: compiled
+execution_units:
+    users-api:
+        type: eks
+        helm_chart_options:
+            install: true
+    users-store:
+        type: eks
+        helm_chart_options:
+            install: true
+    user-info:
+        type: eks
+        helm_chart_options:
+            install: true
```

Comment:
if this is going in regualr samples we need to make it single exec, but also likely want the multi exec in the pro samples